act-rules-format.md

```diff
@@ -160,26 +160,38 @@ Refer to the [Rule Type](#rule-types) section for detailed definitions of the ru
 Accessibility Requirements Mapping {#accessibility-requirements-mapping}
 ------------------------------------------------------------------------
 
-When an ACT Rule is designed to test conformance to one or more [=Accessibility requirements documents=], the rule <em class="rfc2119">must</em> list all [=accessibility requirements=] from those documents that are not satisfied when one or more of the [=outcomes=] of the rule is `failed`. For example, when designing a rule for WCAG that tests if image buttons have alternative text, the rule maps to success criteria 1.1.1 Non-text content, and 4.1.2 Name, Role, Value. That ACT Rule will list both success criteria in its accessibility requirements mapping.
+When an ACT Rule is designed to test conformance to one or more [=Accessibility requirements documents=], the rule <em class="rfc2119">must</em> list all [=accessibility requirements=] from those documents that are not satisfied when one or more of the [=outcomes=] of the rule is `failed`. The rule <em class="rfc2119">may</em> list accessibility requirements that could be not satisfied when the rule outcome is failed. There are two types of accessibility requirements:
```

Comment:
That's about the related one, e.g. Scenario 4: A rule to test 2.4.4 by checking that links have non-empty name could also trigger failure of 1.1.1 on `area`. We do not want to mark an implementation as non-consistent because it is flagging 1.1.1 in some cases, and it could make sense to do so for some implementation (e.g.RGAA is testing `area` elements separately). But this is not a conformance requirement since there is no implication "rule fail => 1.1.1 not satisfied".

The "may" part is a bit of a catch-all (e.g., the same rule could also be mapped to 1.4.1 under the same reasoning that some links are only distinguished by color and sometimes would be failing both). This would be for rule writers to decides (and for example, consider unlikely that an implementation would report all the failure of a link together, while it is likely that it would report both 2.4.4 and 1.1.1 on an image link without name; so 1.1.1 should be a secondary mapping while 1.4.1 shouldn't).
That's a bit too loose to my taste, but I'm afraid we can't do much better in this document. This kinda says "rule writter may add secondary requirement as they see fit" 🤷

Reply:
```suggestion
When an ACT Rule is designed to test conformance to one or more [=Accessibility requirements documents=], the rule <em class="rfc2119">must</em> list all [=accessibility requirements=] from those documents that are not satisfied when one or more of the [=outcomes=] of the rule is `failed`. The rule <em class="rfc2119">may</em> list accessibility requirements that might be not satisfied when the rule outcome is failed. There are two types of accessibility requirements:
```

Changing just this word makes a lot of difference for me. Would it be okay?